packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -99,138 +90,122 @@ function ChangelogTooltip<TRef extends HTMLElement>({
-              <div className="flex gap-4 py-2 px-3 mt-4 font-bold w text-theme-color-salt typo-footnote">
-                <span
-                  className="flex gap-2"
-                  data-testid="changelogUpvotesCounter"
-                >
-                  <UpvoteIcon secondary />
-                  <InteractionCounter value={post.numUpvotes} />
-                </span>
-                <span
-                  className="flex gap-2"
-                  data-testid="changelogCommentsCounter"
-                >
-                  <CommentIcon secondary />
-                  <InteractionCounter value={post.numComments} />
-                </span>
-              </div>
-            </section>
-            <footer className="flex gap-3 justify-between items-center py-3 px-4 w-full h-16 border-t border-theme-divider-tertiary">
-              <Button
-                className="btn-tertiary"
-                onClick={dismissChangelog}
-                tag="a"
-                href={post.commentsPermalink}
-                data-testid="changelogReleaseNotesBtn"
-              >
-                Release notes
-              </Button>
-              {isExtension && (
-                <Button
-                  tag={isFirefoxExtension ? 'a' : undefined}
-                  href={
-                    isFirefoxExtension ? updateFirefoxExtensionLink : undefined
-                  }
-                  className="bg-cabbage-40 btn-primary"
-                  data-testid="changelogExtensionBtn"
-                  loading={isExtensionUpdating}
-                  onClick={onExtensionUpdateClick}
-                >
-                  Update extension
-                </Button>
-              )}
-            </footer>
+            </Button>
           </div>
-        )
-      }
-      offset={tooltipArrowOffset}
-      interactive
-      container={{
-        className: 'shadow',
-        paddingClassName: 'p-0',
-        roundedClassName: 'rounded-16',
-        bgClassName: 'bg-cabbage-40',
-        ArrowComponent: ({ className, ...arrowProps }) => {
-          return (
-            <div
-              {...arrowProps}
-              className={classNames(
-                'flex items-center h-2 -z-1',
-                sidebarExpanded ? '-left-14' : '-left-[2.65rem]',
-              )}
+        </section>
+        <footer className="flex gap-3 justify-between items-center py-3 px-4 w-full h-16 border-t border-theme-divider-tertiary">
+          <Button
+            className="btn-tertiary"
+            onClick={dismissChangelog}
+            tag="a"
+            href={post.commentsPermalink}
+            data-testid="changelogReleaseNotesBtn"
+          >
+            Release notes
+          </Button>
+          {isExtension && (
+            <Button
+              tag={isFirefoxExtension ? 'a' : undefined}
+              href={isFirefoxExtension ? updateFirefoxExtensionLink : undefined}
+              className="bg-cabbage-40 btn-primary"
```

Comment:
I see this was already there, but it should actually be 
```suggestion
              className="btn-primary-cabbage"
```